packages/button/src/vaadin-button.js

```diff
@@ -98,13 +98,13 @@ class Button extends ButtonMixin(ElementMixin(ThemableMixin(ControllerMixin(Poly
         }
       </style>
       <div class="vaadin-button-container">
-        <span part="prefix">
```

Comment:
Note if someone comes around this with the question: 

But isn't this a problem for icon-only buttons? 

For accessible icon-only buttons it was already mandatory to add an `aria-label` to the `vaadin-button` element.

Reply:
Also, for icon-only buttons, it's still possible to place the icon inside a default slot (at least in the web component).

Reply:
Indeed, providing an accessible name for the icon itself has worked before, but the recommended approach has always been to set `aria-label` on the button instead.